addons/mail/static/src/drag_and_drop/dropzone/dropzone.js

```diff
@@ -0,0 +1,59 @@
+/** @odoo-module **/
+
+import { Component, onMounted, useExternalListener, useRef, useState } from "@odoo/owl";
+
+
+export class Dropzone extends Component {
+    static template = "mail.dropzone";
+    static props = { ref: Object };
+
+    setup() {
+        // Prevents to browser to open or download the file when it is dropped
+        // outside of the dropzone.
+        useExternalListener(window, "dragover", ev => ev.preventDefault());
+        useExternalListener(window, "drop", ev => ev.preventDefault());
+
+        this.root = useRef("root");
+        this.state = useState({ isDraggingInside: false });
+
+        this._dragCount = 0;
```

Comment:
Would just remove the `_` prefix, because it's old guidelines and the code is more readable without this.